javascript/javascript1/week1/houseyPricey.js

```diff
@@ -0,0 +1,21 @@
+
+/*
+a house that is 8m wide, 10m deep and 10m high. 
```

Comment:
Check the assignment description again. There's two different houses that has been bought at two different prices ;)

Reply:
😄 just read it again
modified the whole code and pushed all again

thanks.